### DISCUSSION TITLE
Production

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1875] [General]
Hello.

I did the steps that come in the video to make a production space, but in the end the master branch does not appear in the repository.

I already checked the video again and I think I'm not missing anything.
This is what appears in the production repository

![image](https://user-images.githubusercontent.com/117297044/201781463-e20ec79f-7f46-45a5-a7c9-694fcfaae3d8.png)

![image](https://user-images.githubusercontent.com/117297044/201781589-5b9f80b6-3004-42e7-9093-f8f4d9cd807c.png)

Can you help me with this please?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz I am wondering if the video is out of date. Github has started using "main" instead of "master". Can you try continuing to see if the "main" branch will work?

**Comment by SandraEliz** [comment]
Hello

So I do the steps in the video but with the main branch?
or do I only select the main branch in production?

**Reply by brockfanning** [reply]
As far as I understand - you do not need to change anything in your staging repositories, as long as [this line](https://github.com/open-sdg/open-sdg-site-starter/blob/develop/.github/workflows/deploy-to-production.yml#L49) refers to the correct branch name in your production repository.

For example, if that line says `BRANCH: main` then that means your production repository should have a "main" branch. But if that line says `BRANCH: master` then your production repository should have a "master" branch.

Since your production repository only has a "main" branch, I would recommend that you ensure that line says `BRANCH: main`. Then you can continue with the process.

**Reply by SandraEliz** [reply]
Hello

I checked the line you mention and it has main.

When passing changes to productive they are not reflected.
Am I missing any other settings?

**Comment by SandraMarq** [comment]
Hi.

The production site is already working.

Thank you very much
